NOMENCLATURE.md

```diff
@@ -0,0 +1,56 @@
+# Glossary and Nomenclature - General
+
+## Aztec Labs
+The company - aztec.network
+
+## Aztec Protocol
+Definition of a layer 2 network that rolls up to Ethereum. Includes how to execute public/private functions, verify proofs, and reaches consensus on shared state.
+
+## Aztec Network
+Computers on a network that adhere to the Aztec Protocol.
+
+## Aztec Client Software
+Software run on individual computers that adhere to the Aztec protocol.
+(learn more from the Yellow Paper)
+
+## Aztec (Public) Virtual Machine
+Executes the public section of a transaction, and unconstrained functions.
```

Comment:
unconstrained functions are executed by the PXE, client-side, and are just read operations